techniques/general/G204.html

```diff
@@ -115,21 +112,21 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
          </ul>
       
    </section>
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../css/C20">C20: Using relative measurements to set column widths so that lines can average 80 characters or less when the browser is resized</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Open the content that contains a block of text in a common user agent. </li>
-			<li>Check to see if the user agent has a setting that needs to be enabled to allow for reflow, and if so, enable it.</li>
-            <li>Narrow the viewing window to 1/4 of the screen width. </li>
-            <li>Check to see that the content does not require horizontal scrolling to read a line of text. </li>
+            <li>一般的なユーザエージェントで、テキストブロックを含むコンテンツを開く。</li>
+			<li>ユーザエージェントにリフローを許可する設定があるか確認し、もしあれば有効にする。</li>
+            <li>ウィンドウ幅をスクリーン幅の 1/4 に縮める。</li>
+            <li>1 行の文章を読むために、コンテンツを横スクロールする必要がないことを確認する。</li>
```

Comment:
Check to see を「チェックする」としました。
```suggestion
            <li>1 行の文章を読むために、コンテンツを横スクロールする必要がないことをチェックする。</li>
```